src/localconf.js

```diff
@@ -298,6 +300,9 @@ function buildConfig(opts) {
   const { workspacePageTitle } = config;
   const { workspacePageDescription } = config;
 
+  const workspaceRegistrationConfig = resourceRegistrationConfig ? resourceRegistrationConfig.workspaceRegistrationConfig : null;
+  const kayakoConfig = resourceRegistrationConfig ? resourceRegistrationConfig.kayakoConfig : null;
```

Comment:
Probably need to think about how to better organize fields in config files so this config can be shared by both ws reg and study reg

Reply:
I would love to discuss about that with you. I was trying to bring a common config which forms the base for everything that is related to resource registration in general. Within which I added workspaceRegistration and KayakoConfig, Tried to add studyRegistrationConfig as well, but decided that would be like making too many changes in a single PR.

Reply:
I was thinking something like: 

{
 registrationConfigs: {
    features: [
       studyRegistration: {
         // study reg configs...
       },
       workspaceRegistration: {
         // workspace reg configs...
       },
       ... // other reg configs in the future
    ],
    kayakoConfig: {
      // Kayako configs...
    }
}

Of course this needs polish. But if we can do this, then we can put more breaking changes into one release, and bumping up the major version only once

Reply:
Absolutely. Let me go ahead and test this config and make a new commit.

Reply:
I did change the config based on your suggestion and ran some tests in qa-brh and qa-heal. The new changes are working really well. When I publish changes to gitops-qa and cdis-manifest with the new changes, I'll make sure I'll add heal's changes as well.

Reply:
Please update the design doc as well as the description of this PR to reflect what changes has been made for portal config. For PR description, they should be either in `Beaking changes` or `Deployment changes`

Also update this file https://github.com/uc-cdis/data-portal/blob/master/docs/portal_config.md#the-portal-config-file
If you feel you need to write sore explanatory texts, you can also create a dedicated markdown file in here https://github.com/uc-cdis/data-portal/tree/master/docs